src/app/screens/Payment/index.js

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect } from 'react';
+import { connect } from 'react-redux';
+import { UTCard, UTLabel, UTLoading } from '@widergy/energy-ui';
+import i18next from 'i18next';
+import { objectIsEmpty } from '@widergy/web-utils/lib/object';
+import { push } from 'connected-react-router';
+
+import PaymentsActions from 'redux/payments/actions';
+import { paymentType } from 'types/paymentTypes';
+import { PAYMENTS_HISTORY } from 'constants/routes';
+
+import styles from './styles.module.scss';
+
+const Payment = ({ payment = {}, dispatch }) => {
```

Comment:
```suggestion
const Payment = ({ selectedPayment = {}, dispatch }) => {
```
Por ejemplo, para aportar algo de semántica.

Reply:
Cambiado!